packages/widget_toolkit_biometrics/README.md

```diff
@@ -0,0 +1,154 @@
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|  
+| <img src="https://github.com/Prime-Holding/widget_toolkit/blob/77aae47366405628e22ba478fafca1cb41120d19/packages/widget_toolkit_biometrics/doc/success_light.gif?raw=true" alt="Enable/Disable"></img> | <img src="https://github.com/Prime-Holding/widget_toolkit/blob/77aae47366405628e22ba478fafca1cb41120d19/packages/widget_toolkit_biometrics/doc/not_supported_light.gif?raw=true" alt="Not Supported"></img> | <img src="https://github.com/Prime-Holding/widget_toolkit/blob/77aae47366405628e22ba478fafca1cb41120d19/packages/widget_toolkit_biometrics/doc/not_setup_light.gif?raw=true" alt="Not Setup"></img> |
+|                                                                                                                                                                                                        |                                                                                                                                                                                                             |                                                                                                                                                                                                     |
+| <img src="https://github.com/Prime-Holding/widget_toolkit/blob/77aae47366405628e22ba478fafca1cb41120d19/packages/widget_toolkit_biometrics/doc/success_dark.gif?raw=true" alt="Enable/Disable"></img>  | <img src="https://github.com/Prime-Holding/widget_toolkit/blob/77aae47366405628e22ba478fafca1cb41120d19/packages/widget_toolkit_biometrics/doc/not_supported_dark.gif?raw=true" alt="Not Supported"></img>  | <img src="https://github.com/Prime-Holding/widget_toolkit/blob/77aae47366405628e22ba478fafca1cb41120d19/packages/widget_toolkit_biometrics/doc/not_setup_dark.gif?raw=true" alt="Not Setup"></img>  |
```

Comment:
Make sure that all essential recordings are added in the pubspec as well

See an example here: https://github.com/Prime-Holding/widget_toolkit/blob/772fae8ea3e625d1550ca3bd4fea788161fe156a/packages/widget_toolkit/pubspec.yaml#L71